tests/core/engine_adapter/test_base_postgres.py

```diff
@@ -2,10 +2,13 @@
 import typing as t
 from unittest.mock import call
 
+import pytest
 from sqlglot import exp, parse_one
 
 from sqlmesh.core.engine_adapter.base_postgres import BasePostgresEngineAdapter
 
+pytestmark = [pytest.mark.postgres, pytest.mark.redshift]
```

Comment:
Shouldn't this include `pytest.mark.engine` since `engine` is:

> engine: test all engine adapters

? Eg. if I run with `-m engine` I'd expect all engine tests to run but it will actually only run the common logic. Maybe rename to `engine_common`?

Reply:
Yep thanks